_dprhtml/js/dict_xml.js

```diff
@@ -678,6 +678,10 @@ function displayDictData(data) {
     $(difbId).append(dataNode);
     $('#paliTextContent').scrollTop(0);
   }
+
+  if (DPR_PAL.isNavigationFeature()) {
+    __otherDialogsViewModel.showBottomPane(BottomPaneTabsViewModel.TabNumber.DICTIONARY);

```

Comment:
Originally, I used:
```javascript
__otherDialogsViewModel.showBottomPane(1);
```
but the value `1` is quite opaque, so I wanted to make it much easier for future maintainers to understand what this line is doing at a glance. Though it was not strictly necessary to create the [BottomPaneTabsViewModel.TabNumber](https://github.com/digitalpalireader/digitalpalireader/pull/330/files#diff-5f5df7e136837b52a20b4c9ce92df2008224a9b3d6aa4ab0a02e693ef1c7734aR4-R10) enum, I think the enhanced readability is worth the trade-off.

Reply:
@parthopdas Ah, I understand now, thank you! Yes, using DPR1_chrome_mod.DPRShowBottomPane() does work! I will make that change.